### ISSUE TITLE
a11y CRITICAL: `auro-radio` color contrast on error message (Hawaiian theme only)

### ISSUE THREAD

**Issue by sun-mota** [AlaskaAirlines/auro-formkit#625]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [ ] Desktop
- [ ] Mobile

### Steps to reproduce

![Image](https://github.com/user-attachments/assets/22649735-16a0-47e8-812c-55e08bd9e0d2)

current value: var(--ds-basic-color-status-error)

  Error: Accessibility Violations
      ---
      Rule: color-contrast
      Impact: serious
      Elements must meet minimum color contrast ratio thresholds (https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=axeAPI)
      
      Issue target: auro-radio-group,auro-formkit-radio-helptext-1_0_0,.helptext-wrapper
      Context: <div class="helptext-wrapper" visible="">
                <slot></slot>
            </div>
      Fix any of the following:
        Element has insufficient color contrast of 4.31 (foreground color: #e53339, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
      ---

### Expected behavior

Expected contrast ratio of 4.5:1

### Reproducible on the Auro doc site

- [ ] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by sun-mota** [comment]
aria-disabled should be added to indicate that the element is disabled.

**Comment by sun-mota** [comment]
confirmed that it's resolved in the latest layoutSupport
